notifier/settings.py

```diff
@@ -193,6 +193,8 @@
 # before it's replaced with a new one.
 CELERYD_MAX_TASKS_PER_CHILD = int(os.getenv('CELERYD_MAX_TASKS_PER_CHILD', 100))
 
+ROUTING_KEY = os.getenv('NOTIFIER_ROUTING_KEY', '')
```

Comment:
Should the default be ```''``` or ```None```? It should probably be something sensible or something that makes it not run so that it's obvious it hasn't been set. I'm not sure what the right thing is here. I could use a second opinion.

Reply:
@jdmulloy  I agree, we should not set the routing key to `''`  @syed-awais-ali  if this setting is set to none what happens? I assume it uses celery's default?